index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<link rel="stylesheet" href="styles.css"></link>
+<meta charcode="UTF-8">
+<title>Survey Form
```

Comment:
Kindly prefer to use a more descriptive title that would improve your websites accessibility.